src/main/java/io/zrz/jpgsql/client/opj/PgResultRows.java

```diff
@@ -232,8 +232,7 @@ public Collection<String> textArray(int row, int column) {
 
     }
 
-    throw new IllegalArgumentException();
-
+    throw new IllegalArgumentException(String.format("unhandled format for text array field: %s", field.format()));
```

Comment:
❤️

Reply:
nit: https://github.com/auth0/jpgsql-client/blob/36ea4b698cb985bea5ca503360a0025ec2e6b636/src/main/java/io/zrz/jpgsql/client/opj/PgResultRows.java#L213. could you do the same log

Reply:
actually, I will log the whole field (which includes the value of format):

```
  public String toString() {
    final StringBuilder sb = new StringBuilder();
    sb.append(this.label());
    sb.append("[").append(this.col).append(":").append(this.position()).append("]");
    sb.append(" = { ");
    sb.append("oid:").append(this.oid()).append(" ");
    sb.append("tbl:").append(this.tableoid()).append(" ");
    sb.append("len:").append(this.length()).append(" ");
    sb.append("fmt:").append(this.format()).append(" ");
    sb.append("mod:").append(this.modifier()).append(" ");
    sb.append("}");
    return sb.toString();
  }
```